modules/fts/pages/fts-supported-queries-boolean-query.adoc

```diff
@@ -0,0 +1,27 @@
+[#boolean-query]
+= Boolean Query
+
+A boolean query is a compound query that allows for the user to employ a combination of conjunction and disjunction queries.
+It takes three groups of queries:
+
+* *must*: Result documents must satisfy all of these queries
+
+* *must_not*: Result documents should satisfy any of these queries
+
+* *should*: Result documents must not satisfy any of these queries
```

Comment:
Are the descriptions of "should" and "must not" swapped?

Separate note: a disjunction query's "min" property works for both "should" and "must not", which lets a user express things like "MUST NOT satisfy [min] or more of these disjuncts", or "SHOULD satisfy at least [min] of these disjuncts". I don't know if it's worth documenting that here, but I found it interesting.

Reply:
Shoot thanks.

Ya should look into updating the min part separately.